bricks/template/test.cc

```diff
@@ -274,3 +276,89 @@ TEST(TemplateMetaprogrammingInternalTest, NonemptyConstructorForCombiner) {
   EXPECT_EQ("foo", joined("test"));
   EXPECT_EQ(42, joined(0));
 }
+
+// ********************************************************************************
+// * `current::decay_t` test, moved here from `decay.h`.
+// ********************************************************************************
+template <typename A, typename B>
+inline constexpr bool is_same_v = std::is_same_v<A, B>;
```

Comment:
I remember! This is the line that did not compile in C++14 on Windows, and I just used `std::is_same_v`.